docs/AI-for-security/attack-discovery.asciidoc

```diff
@@ -45,18 +53,18 @@ When you access Attack discovery for the first time, you'll need to select an LL
 .Recommended models
 [sidebar]
 --
-While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3.5 Sonnet. In general, models with larger context windows are more effective for Attack discovery.
+While Attack discovery is compatible with many different models, refer to the <<llm-performance-matrix, Large language model performance matrix>> to see which models perform best.
 --
 +
-image::images/attck-disc-select-model-empty-state.png[]
+image::images/attck-disc-select-model-empty.png[]
 +
 . Once you've selected a connector, click **Generate** to start the analysis.
 
 It may take from a few seconds up to several minutes to generate discoveries, depending on the number of alerts and the model you selected. 
 
-IMPORTANT: Attack discovery is in technical preview and will only analyze opened and acknowleged alerts from the past 24 hours. By default it only analyzes up to 20 alerts within this timeframe, but you can expand this up to 100 by going to **AI Assistant → Settings (image:images/icon-settings.png[Settings icon,17,17]) → Knowledge Base** and updating the **Alerts** setting.
+IMPORTANT: By default it analyzes up to 100 alerts within this timeframe, but you can expand this up to 500 by clicking the settings icon  (image:images/icon-settings.png[Settings icon,17,17]) next to the model selection menu and adjusting the **Alerts** slider. Note that sending more alerts than your chosen LLM can handle may result in an error.
```

Comment:
```suggestion
IMPORTANT: By default, attack discovery analyzes up to 100 alerts within this timeframe, but you can expand this up to 500 by clicking the settings icon  (image:images/icon-settings.png[Settings icon,17,17]) next to the model selection menu and adjusting the **Alerts** slider. Note that sending more alerts than your chosen LLM can handle may result in an error.
```

Since the first sentence is deleted, should "within this timeframe" say "from the past 24 hours"? Or does it no longer have this time constraint?